frontend/components/common/PrerenderedHTML.tsx

```diff
@@ -313,26 +344,42 @@ export const styles: SystemStyleObject = {
          paddingInline: 3,
       },
    },
+};
 
-   '.fa-svg-icon': {
-      svg: {
-         width: '1em',
-         height: '1em',
-         display: 'inline-block',
-
-         path: {
-            fill: 'currentColor',
-         },
-      },
-
-      '&.xs svg': {
-         width: '0.5em',
-         height: '0.5em',
-      },
-
-      '&.sm svg': {
-         width: '0.75em',
-         height: '0.75em',
-      },
-   },
+const commerceStyles: SystemStyleObject = {
+   ...sharedStyles,
 };
+
+export const PrerenderedHTML = chakra(function Prerendered({
+   html,
+   styles,
+   className,
+}: {
+   html: string;
+   styles: 'troubleshooting' | 'commerce';
+   className?: string;
+}) {
+   useEffect(() => {
+      import('lite-youtube-embed');
+   }, []);
```

Comment:
Minor: It would be cool to only include this import if it is required. It's unclear if this is possible to determine

Reply:
That's a good point - commerce will rarely use it, if ever. I have an idea

Reply:
Can I get a sanity check on this before I commit? It involves renaming the prop `style` to `template`, but it's way more understandable from a readability perspective:

<details>
<img width="1557" alt="Screenshot 2023-09-06 at 4 05 20 PM" src="https://github.com/iFixit/react-commerce/assets/1634505/2c410b13-a4bc-4b5b-b8b5-24837522ecd6">

</details>

Reply:
I think `template` is a reasonable name. Chakra uses the name `variant` for things like this, but "variant" is already overloaded in our app (product variants, topic variants)

Reply:
[`045ad14`](https://github.com/iFixit/react-commerce/pull/1946/commits/045ad145baeea38012685c4777b1909025e0f046) and [`c7be0a1`](https://github.com/iFixit/react-commerce/pull/1946/commits/c7be0a133e21037a08203d32f3d46e31b41df43e)